cilium-health/launch/endpoint.go

```diff
@@ -304,7 +304,7 @@ func LaunchAsEndpoint(baseCtx context.Context,
 
 	pidfile := filepath.Join(option.Config.StateDir, PidfilePath)
 	prog := "ip"
-	args := []string{"netns", "exec", netNSName, binaryName, "--pidfile", pidfile}
+	args := []string{"netns", "exec", netNSName, binaryName, "--listen", strconv.Itoa(option.Config.ClusterHealthPort), "--pidfile", pidfile}
```

Comment:
For cross-reference: https://github.com/cilium/cilium/blob/aef5002779c7842f23884588c13975879489bf14/cilium-health/responder/main.go#L40